src/index.html

```diff
@@ -4,13 +4,36 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Webpack Exercise</title>
+    <title>Leaderboard</title>
     <link rel="apple-touch-icon" sizes="180x180" href="assests/favicon/apple-touch-icon.png">
     <link rel="icon" type="image/png" sizes="32x32" href="assests/favicon/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="assests/favicon/favicon-16x16.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
     <link rel="manifest" href="assests/favicon/site.webmanifest">
 </head>
 <body>
-    <h1>Hello Webpack</h1>
+    <header>
+        <h1 class="header">Leaderboard</h1>
+    </header>
+    <main>
+        <section class="recent-score">
+            <div>
+                <h2>Recent Scores</h2>
+                <button type="submit" class="refresh-btn">Refresh</button>
+            </div>
+            <ul class="recent-score-list">                  
+            </ul>
+        </section>
+        <section class="add-score">
+            <h2>Add your score</h2>
+            <form class="add-score-form">
+                <input type="text" name="name" placeholder="Your name" required>
+                <input type="number" name="score" placeholder="Your score" required>
+                <button type="submit">Submit</button>
+            </form>
+        </section>
+    </main>
 </body>
```

Comment:
- [ ] Great job so far 👏🏻 However, According to the project requirements, you should write plain HTML markup with minimum styling. Kindly make sure your HTML code looks like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), styling is not required for this project


![image](https://user-images.githubusercontent.com/79058364/195183007-4bf728d2-75a0-4782-8c49-e57eaa2048e3.png)

- Yours is empty:
![Screenshot from 2022-10-11 21-39-42](https://user-images.githubusercontent.com/79058364/195183570-dd8b34d7-2fa9-4ee1-8b9c-4174b9bcf250.png)